setup.py

```diff
@@ -75,6 +75,7 @@ def get_version():
     version=get_version(),
     description='OVOS skill plugin',
     long_description=long_description,
+    long_description_content_type='text/markdown',
     url=URL,
     author='Aix',
     author_email='aix.m@outlook.com',
```

Comment:
Future: since he isn't maintaining this, should this be changed to an OVOS email?

Reply:
we need a org wide sweep to move to @openvoiceos.org emails and to change copyright in license files